From b2a2c4c8406b0fe22280ffea4216ad0b31a07967 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Fri, 20 Sep 2024 01:44:53 +0200
Subject: [PATCH] ges: Fix name of GESFrameCompositionMeta API type

g-ir-scanner 1.82.0 failed with a fatal error due to the wrong
namespace.

Fixes: https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/3800
Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/7556>
---
 .../gst-editing-services/ges/ges-frame-composition-meta.c      | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/subprojects/gst-editing-services/ges/ges-frame-composition-meta.c b/subprojects/gst-editing-services/ges/ges-frame-composition-meta.c
index d8c5a5251309..1ad7b6cfe4c7 100644
--- a/subprojects/gst-editing-services/ges/ges-frame-composition-meta.c
+++ b/subprojects/gst-editing-services/ges/ges-frame-composition-meta.c
@@ -47,7 +47,8 @@ ges_frame_composition_meta_api_get_type (void)
   static const gchar *tags[] = { "video", NULL };
 
   if (g_once_init_enter (&type)) {
-    GType _type = gst_meta_api_type_register ("GstFrameCompositionApi", tags);
+    GType _type =
+        gst_meta_api_type_register ("GESFrameCompositionMetaAPI", tags);
     g_once_init_leave (&type, _type);
   }
   return type;
-- 
GitLab

